include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -554,6 +572,25 @@ class argument_parser
     //!\brief Stores the sub-parser in case \link subcommand_arg_parse subcommand parsing \endlink is enabled.
     std::unique_ptr<argument_parser> sub_parser{nullptr};
 
+    /*!\brief The format of the argument parser that decides the behavior when
+     * calling the seqan3::argument_parser::parse function.
+     * \details The format is set in the function argument_parser::init.
+     */
+    std::variant<detail::format_parse,
+                 detail::format_help,
+                 detail::format_short_help,
+                 detail::format_version,
+                 detail::format_html,
+                 detail::format_man,
+                 detail::format_copyright/*,
+                 detail::format_ctd*/> format{detail::format_help{{}, false}}; // Will be overwritten in any case.
+
+    //!\brief List of option/flag identifiers that are already used.
+    std::set<std::string> used_option_ids{"h", "hh", "help", "advanced-help", "export-help", "version", "copyright"};
```

Comment:
also this is the only place where these entities are set/used?

Reply:
yes